src/components/FormElements/AssetPicker.tsx

```diff
@@ -1,4 +1,6 @@
-import { useState } from "react";
+/* eslint-disable */
```

Comment:
Let's replace these with more specific rules. Inline if possible, or global for a specific rule.